app/Http/Requests/Projects/UpdateRequest.php

```diff
@@ -38,5 +37,16 @@ public function rules()
             'customer_id'    => 'nullable|numeric',
             'status_id'      => 'required|numeric',
         ];
+
+        //Allow for flexibility instead of optionless hard-coded value for "description". This is
+        //achieved using environmental variable.
+        //A value of zero (0) will mean "no limit"
+
+        $char_len_project_description = intval(env('CHAR_LEN_PROJECT_DESCRIPTION', 255));
```

Comment:
Change this to **DESCRIPTION_CHAR_LEN**, too. And change variable name to **camelCase**.